specs/protocol/tx_format.md

```diff
@@ -170,15 +170,20 @@ Creates a contract with contract ID as computed [here](./identifiers.md#contract
 
 ## TransactionMint
 
-| name               | type                    | description                              |
-|--------------------|-------------------------|------------------------------------------|
-| `outputsCount`     | `uint8`                 | Number of outputs.                       |
-| `outputs`          | [Output](#output)`[]`   | List of outputs.                         |
+The transaction is created by the block producer and is not signed.
+It's not usable outside of block creation or execution, so all fields must be fully set upon creation without any zeroing.
```

Comment:
```suggestion
The transaction is created by the block producer and is not signed. Since it is not usable outside of block creation or execution, all fields must be fully set upon creation without any zeroing.
```